userinterface/flyui.ui

```diff
@@ -32,6 +32,22 @@
    <property name="bottomMargin">
     <number>0</number>
    </property>
+   <item>
+    <widget class="QGroupBox" name="groupBox">
+     <property name="title">
+      <string>Onboard Computer</string>
```

Comment:
Would suggest "Flight Controller" in this case to avoid confusion with an additional RPi or Jetson type computer

Reply:
I though about that too. Not sure what to name it, because it will retrieve parameters from both flight controller and onboard computer (if there is one)...